callosum/ordering.py

```diff
@@ -61,7 +61,7 @@ class AbstractAsyncScheduler(metaclass=abc.ABCMeta):
         raise NotImplementedError
 
     @abc.abstractmethod
-    async def wait(self, request_id):
+    async def get_fut(self, request_id):
```

Comment:
Could you describe why you change this name?

Reply:
First reason: it seemed to me that the only functionality of this method is to return `asyncio.Future` object. It is **not** using dict.get(), but rather looks up assuming that the object exists in dictionary and raises error otherwise, apparently. 

Second reason: I think that using "wait" may be misleading since we do not wait (or await) but rather return Future.

The second reason above is not quite convincing, though. However, I also use this method in other part of code like:
`fut = self.get_fut(rqst_id)` 
in line 115 in _ordering.py_. So I thought making more general name for this method would be more descriptive, when using it for various purposes and not only for subsequent awaiting.

Does it justify renaming?